include/klee/Config/Version.h

```diff
@@ -21,6 +21,12 @@
 #  define LLVM_TYPE_Q const
 #endif
 
+#if LLVM_VERSION_CODE >= LLVM_VERSION(3, 7)
+#  define KLEE_LLVM_GEP_INST_1ST(x) (x),
```

Comment:
I don't really like the name of the macro, since it doesn't really say what it's for. I suggest `KLEE_LLVM_GEP_TYPE` or something similar, since the first argument is pointeeType.
@jirislaby What do you say?